### DISCUSSION TITLE
dont use fsr 3.1 frame gen

### DISCUSSION THREAD

**Discussion by octanejohn** [godotengine/godot-proposals#10453] [3D]
wait for extrapolation ideas to mature or maybe not demand neural network hardware

https://poiw.github.io/ExtraSS
https://arxiv.org/abs/2406.18551

**Comment by Calinou** [comment]
Rendering papers tend not to mention the downsides of the techniques exposed within, so I'd wait until this lands in production first.

Either way, FSR 3.1 is open source already.

**Reply by octanejohn** [reply]
ye as i said more mature also frame gen needs anti-lag/+ or maybe even the anti-lag vulkan extension to have usable latency
